src/documentation/markdown.rs

```diff
@@ -71,15 +71,47 @@ impl<'a> PlainOverlay<'a> {
                                 // @todo validate as if it was another document entity
                             }
                         }
+                        Tag::Link(link_type, _url, title) => {
+                            // @todo check links
+                            // for now, only dealing with some links types
+                            match link_type {
+                                LinkType::Inline => {}
+                                LinkType::Autolink | LinkType::Email => skip_text = true,
+                                //Reference,
+                                //ReferenceUnknown,
+                                //Collapsed,
+                                //CollapsedUnknown,
+                                //Shortcut,
+                                //ShortcutUnknown,
```

Comment:
I think we should have a test case for each of them, and then enable them one by one and see if special handling is necessary.

Reply:
yes, I agree. I will look into this and get back to you.